vscode/README.md

```diff
@@ -254,28 +254,117 @@ These are the settings that may impact the Ruby LSP's behavior and their explana
 
 ### Multi-root workspaces
 
+Multi-root workspaces are VS Code's way to allow users to organize a single repository into multiple distinct concerns.
```

Comment:
Since we already decided to stick with VS Code's multi-root workspace approach, I think it's worth emphasize this here. Maybe with something like:

```md
> ![NOTE]
> To make sure Ruby LSP works well with your multi-root workspace project, please
> read through the instructions below and configure it following the examples
```